specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/agreement.json

```diff
@@ -67,215 +64,274 @@
-          "description": "The list of billing profiles associated with agreement and present only for specific agreements.",
+          "type": "string",
           "readOnly": true
         },
-        "category": {
-          "description": "The category of the agreement signed by a customer.",
-          "enum": [
-            "MicrosoftCustomerAgreement",
-            "AffiliatePurchaseTerms",
-            "Other"
-          ],
+        "billingProfilesInfo": {
+          "description": "The list of billing profiles associated with agreement and present only for specific agreements.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BillingProfileInfo"
+          },
           "readOnly": true,
+          "x-ms-identifiers": []
+        },
+        "displayName": {
+          "description": "The name of the agreement signed by a customer.",
           "type": "string",
-          "x-ms-enum": {
-            "modelAsString": true,
-            "name": "category"
-          }
+          "readOnly": true
         },
         "effectiveDate": {
-          "description": "The date from which the agreement is effective.",
           "format": "date-time",
-          "readOnly": true,
-          "type": "string"
+          "description": "The date from which the agreement is effective.",
+          "type": "string",
+          "readOnly": true
         },
         "expirationDate": {
-          "description": "The date when the agreement expires.",
           "format": "date-time",
-          "readOnly": true,
-          "type": "string"
+          "description": "The date when the agreement expires.",
+          "type": "string",
+          "readOnly": true
         },
         "participants": {
           "description": "The list of participants that participates in acceptance of an agreement.",
+          "type": "array",
           "items": {
-            "$ref": "#/definitions/Participants"
+            "$ref": "#/definitions/Participant"
           },
-          "type": "array"
+          "readOnly": true,
+          "x-ms-identifiers": []
         },
         "status": {
           "description": "The current status of the agreement.",
+          "enum": [
+            "Unknown",
+            "Draft",
+            "PendingSignature",
+            "Canceled",
+            "Declined",
+            "Active",
+            "Expired",
+            "Terminated",
+            "Deleted",
+            "Submitted"
+          ],
+          "type": "string",
           "readOnly": true,
-          "type": "string"
+          "x-ms-enum": {
+            "name": "AgreementStatus",
+            "modelAsString": true
+          }
+        },
+        "leadBillingAccountName": {
+          "description": "The ID of the lead billing account if this agreement is part of the Customer Affiliate Purchase Terms.",
+          "type": "string",
+          "readOnly": true
         }
       }
     },
     "BillingProfileInfo": {
```

Comment:
Can we just call it BillingProfile? This will be in the Agreement namespace anyway.

Reply:
Keeping as is since it differs from the `BillingProfile` resource.